### PR TITLE
Fix entrypoint

### PR DESCRIPTION
**Pull request by nyan-ko**
Modify factories and controllers so they are initialized with the necessary data to function. Fix Main.java's pvsm entrypoint so the program runs in a clean way with no hacks. Depends on classes/interfaces from #63.

### CODE REVIEW

File: src/main/java/ui/windows/creation/CreationDisplay.java [hunk 1]
```diff
@@ -166,14 +166,9 @@ public void actionPerformed(ActionEvent e) {
             }
         }
         boolean gameCreated = controller.createGameResponse(controllerInput);
-        PlayerGameInputBoundary playerGameInteractor = controller.getPlayerGameInteractor();
-        GameBridge bridge = new PlayerGameController(playerGameInteractor);
-        gameController.setBridge(bridge);
         // If a Game was created (valid input for game players), then set the window to the Game.
         if(gameCreated) {
-            gameController.startGame();
-            layoutManager.setPane(WindowName.GAME);
```
Comment (SMateen21) [inline src/main/java/ui/windows/creation/CreationDisplay.java:175]: Im wondering where  layoutmanager.setPane(GAME) will be called, as to switch the panel to GameDIsplay?

**Review by SMateen21 — COMMENTED** [review]
Looks good, just something small to address. Great job on making this cleaner.

File: src/main/java/ui/windows/creation/CreationDisplay.java (hunk 1 above)
Comment (nyan-ko) [reply src/main/java/ui/windows/creation/CreationDisplay.java:175]: the new navigation button is linked to a panedelegator thatll handle the switch, this method can be cleaned up but i wasnt sure what to change

File: src/main/java/ui/factories/CreatorWindowFactory.java
Comment (meiiph) [inline src/main/java/ui/factories/CreatorWindowFactory.java:1]: There already exists a factory that creates creator/creation windows, I would delete one of them.

File: src/main/java/ui/windows/creation/CreationDisplay.java
Comment (meiiph) [inline src/main/java/ui/windows/creation/CreationDisplay.java:1]: Rename if we choose to stick with the CreatorWindowFactory, for consistency.

File: src/main/java/ui/windows/creation/CreationWindow.java
Comment (meiiph) [inline src/main/java/ui/windows/creation/CreationWindow.java:1]: Rename if we choose to stick with the CreatorWindowFactory, for consistency.

**Review by meiiph — APPROVED** [review]
Looks good to me, just need to figure out consistent naming, and deleting any duplicate files.

**Review by lilyxmeng — APPROVED** [review]

**Review by SMateen21 — APPROVED** [review]
As long as we have a way to switch from the GameCreation screen to the actual game play screen, then it's good to approve.

**Pull request merged**